doc/api/tls.md

```diff
@@ -884,6 +907,17 @@ changes:
     verified against the list of supplied CAs. An `'error'` event is emitted if
     verification fails; `err.code` contains the OpenSSL error code. **Default:**
     `true`.
+  * `pskCallback(hint)` {Function} When negotiating TLS-PSK, this optional
```

Comment:
I am not sure if this is a common format to use a call form with an argument for a function option. Maybe we should use a callback parameter format: just use `` `pskCallback` `` form and add a sublist with `hint {type}` and `* Returns:` entries?